### PR TITLE
[v1.9.x] Remove the "copyright by contributors" line in source files

### PR DESCRIPTION
**Pull request by TristonC**
## Description ##
This PR is the action item 5, for the issue [20475](https://github.com/apache/incubator-mxnet/issues/20475). It removes the "Copyright by Contributors" line in the ./src, ./include, ./tools and 3rdparty/mshadow directories and their sub-directories.

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [x] Remove the "Copyright by Contributors" line in the ./src direcotry and its sub-directories.
- [x] Remove the "Copyright by Contributors" line in the ./include direcotry and its sub-directories.
- [x] Remove the "Copyright by Contributors" line in the ./tools direcotry and its sub-directories.
- [x] Remove the "Copyright by Contributors" line in the ./3rdparty/mshadow direcotry and its sub-directories.

### CODE REVIEW

File: src/operator/numpy/np_einsum_op.cc [hunk 1]
```diff
@@ -34,7 +34,6 @@
  *     with the distribution.
  *
  *  * Neither the name of the NumPy Developers nor the names of any
- *     contributors may be used to endorse or promote products derived
```
[the remark this answers is not in the document]
Comment (TristonC) [reply src/operator/numpy/np_einsum_op.cc:37]: Not sure about this. Should we?

File: src/kvstore/comm.h [hunk 2]
```diff
@@ -18,7 +18,6 @@
  */
 
 /**
```
[the remark this answers is not in the document]
Comment (TristonC) [reply src/kvstore/comm.h]: Done.

File: src/kvstore/gpu_topology.h [hunk 3]
```diff
@@ -18,7 +18,6 @@
  */
 
 /**
```
[the remark this answers is not in the document]
Comment (TristonC) [reply src/kvstore/gpu_topology.h]: Done.

File: src/operator/numpy/np_einsum_op.cc (hunk 1 above)
[the remark this answers is not in the document]
Comment (TristonC) [reply src/operator/numpy/np_einsum_op.cc]: Done.

**Review by josephevans — APPROVED** [review]

**Pull request merged**